testutils/infra/cli.py

```diff
@@ -39,27 +40,40 @@ def __init__(
 
         self.cid = self.container_manager.getid([base_filter])
 
+    def choose_binary_and_config_paths(
+        self, service_flavours: List[str], service_name: str
+    ):
+        """Choose binary and configuration paths depending on service flavour. """
+        for service in service_flavours:
+            try:
+                self.container_manager.execute(self.cid, [service.bin_path, "--help"])
+                self.service = service
+                break
+            except:
+                continue
+        else:
+            raise RuntimeError(f"no runnable binary found in mender-{service_name}")
```

Comment:
`service_name` already has the `mender-` prefix, so this should be:
```suggestion
            raise RuntimeError(f"no runnable binary found in {service_name}")
```

Reply:
done, thanks!